docs/Design/SoftDetailedDes/MIS.tex

```diff
@@ -229,6 +228,89 @@ \subsubsection{Local Functions}
 
 \newpage
 
+\section{MIS of Location Management Module} \label{mLocation} 
+The Location Management Module manages user location data and outputs relevant resources based the available user location data.
+
+\subsection{Module}
+
+locationManagement
+
+\subsection{Uses}
+
+ Database Driver Module (Section \ref{mDBDriver})
+
+\subsection{Syntax}
+
+\subsubsection{Exported Constants}
+None
+
+\subsubsection{Exported Access Programs}
+
+\begin{center}
+\begin{tabular}{p{5cm} p{3cm} p{3cm} p{5cm}}
+\hline
+\textbf{Name} & \textbf{In} & \textbf{Out} & \textbf{Exceptions} \\
+\hline
+getLastLocation & - & String & noLocationSavedException \\
+setLastLocation & String & - & invalidLocationException \\
+getNearbyStores & - & List$<$String$>$ & - \\
+setRadius & $\mathbb{N}$ & - & - \\
+\hline
+\end{tabular}
+\end{center}
+
+\subsection{Semantics}
+
+\subsubsection{State Variables}
+None
+
+\subsubsection{Environment Variables}
+None
+
+\subsubsection{Assumptions}
+None
+
+\subsubsection{Access Routine Semantics}
+
+\noindent getLastLocation():
+\begin{itemize}
+\item transition: none
+\item output: \( \neg location = null \Rightarrow \textit{out} \coloneqq location \), where \textit{location} is the user location stored in the database (Section \ref{mDBDriver})
+\item exception: \( location = null \Rightarrow \textit{exc} \coloneqq noLocationSavedException \), where \textit{location} is the user location stored in the database (Section \ref{mDBDriver})
```

Comment:
\textit{out} should go before the the discrete math stuff.

So it should be `\textit{out} \coloneqq \neg location = null \Rightarrow location` and similar for the exception. `out` and `exc` should be the first things and then the discrete logic afterwards. I'm also gonna assume that you made sure everything compiles before pushing it?